utils/api_info_v2.py

```diff
@@ -28,13 +28,7 @@
 conn_group.add_argument('-p', '--password', help='Password')
 conn_group.add_argument('--host', help='Select another api host (for debugging purposes)')
 
-resp_group = parser.add_argument_group('Response options')
-
-resp_group.add_argument('--raw', default=False, action='store_true', help='Print only raw response (JSON)')
-
-info_group = parser.add_argument_group('Info options')
-
-info_group.add_argument('-q', '--query', required=True, help='Torrent by id or hash Required.')
+parser.add_argument('-q', '--query', required=True, help='Torrent by id or hash Required.')
```

Comment:
This could still be just a `query` with a `metavar='ID_OR_HASH'` - it's the only required argument for the script, so don't make the user type a useless `-q`. It's clear enough what the argument is.